libs/options-api/src/main/java/gov/cdc/nbs/option/jdbc/autocomplete/SQLBasedOptionResolver.java

```diff
@@ -34,6 +63,7 @@ public Collection<Option> resolve(final String keyword, final int limit) {
         CRITERIA_PARAMETER, withWildcard(keyword),
         PREFIX_CRITERIA_PARAMETER, withPrefixWildcard(keyword),
         QUICK_CODE, keyword,
+        RELATED_CLASS_CODES, relatedClassCodes,
```

Comment:
👀 hidden magic options

suggestion:  The related class codes are static and can be added directly to the resulted test query in `ResultedTestOptionsResolver`

Reply:
reusable options.... but yes i can add them directly to the query